src/main/resources/mockCourses_FrontendExpects.json

```diff
@@ -36,7 +36,7 @@
                     "day": "WEDNESDAY"
                 }
             ],
-            "requiredShiftAmount": 2,
+            "requiredShiftAmount": 1,

```

Comment:
As you and I discussed on Slack, these files changes are great, as long as they are consistent across all files. This includes the `..._Full.json` and `..._FrontendExpects.json` files for the tutor, course, shift (if they exist), and schedule mock data json files

Reply:
When I changed it I also made same changes to the other file.